src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -320,6 +326,68 @@ def privacy_request_csv_download(
     return response
 
 
+def get_variable_columns(
+    privacy_request_query: Query,
+) -> tuple[List[str], Dict[str, str]]:
+    identity_columns: Set[str] = set()
+    custom_field_columns: Dict[str, str] = {}
+    for pr in privacy_request_query:
+        get_unique_identity_columns(
+            identity_columns, pr.get_persisted_identity().dict()
+        )
+        get_unique_custom_field_columns(
+            custom_field_columns, pr.get_persisted_custom_privacy_request_fields()
+        )
+
+    return list(identity_columns), custom_field_columns
+
+
+def get_unique_identity_columns(
+    existing_columns: Set[str], identities: dict[str, Any]
+) -> None:
+    for key, value in identities.items():
+        if value:
+            existing_columns.add(key)
+
+
+def get_unique_custom_field_columns(
+    existing_custom_fields: Dict[str, str], custom_fields: dict[str, Any]
+) -> None:
+    for key, value in custom_fields.items():
+        if value["value"]:
+            existing_custom_fields[key] = value["label"]
+
+
+def with_prefix(prefix: str, items: List[str]) -> List[str]:
+    prefixed = []
+    for item in items:
+        prefixed.append(f"{prefix} {item}")
+    return prefixed
```

Comment:
Not a blocker but this could probably be shortened as:
```suggestion
    return [f"{prefix} {item}" for item in items]
```

Reply:
Sure, definitely a pythonism I'm not used to. I'll update since it's idiomatic but I always feel like these statements feel yoda-y.